src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1491,7 +1491,7 @@ private void SetupEditedSpecies(MicrobeSpecies species)
 
         // Disabled warning as this is a tweak constant
         // ReSharper disable ConditionIsAlwaysTrueOrFalse HeuristicUnreachableCode
-        if (Constants.CREATE_COPY_OF_EDITED_SPECIES)
+        if (Constants.CREATE_COPY_OF_EDITED_SPECIES && new Random().NextDouble() > 0.5)
```

Comment:
Probability probably should be a Constant as we may want to tweak in the future, shouldn't it?

Reply:
Yeah, that makes a lot of sense.